backend/events/commands/paragraph.py

```diff
@@ -4,7 +4,11 @@ def __init__(self, ws, node):
         self.node = node
 
     async def emit(self):
-        await self.ws.send_json({"output": self.node.prompt}) 
+        await self.ws.send_json({
+            "output": self.node.prompt,
+            "type": "paragraph",
```

Comment:
```
[{"id":"1c801d8f-2b0c-4cea-a343-b907463aee7e","type":"heading","props":{"textColor":"default","backgroundColor":"default","textAlignment":"left","level":1},"content":[{"type":"text","text":"hello","styles":{"backgroundColor":"red"}}],"children":[]},{"id":"30241821-ac03-4b89-b3dd-2f48fc2c8a9e","type":"paragraph","props":{"textColor":"default","backgroundColor":"default","textAlignment":"left"},"content":[],"children":[]},{"id":"2b2faae9-1b00-494b-84ee-8236af05e531","type":"paragraph","props":{"textColor":"default","backgroundColor":"default","textAlignment":"center"},"content":[{"type":"text","text":"sdf","styles":{"textColor":"orange"}}],"children":[]},{"id":"33c2442b-fe30-43df-9728-d76552c95b30","type":"paragraph","props":{"textColor":"default","backgroundColor":"default","textAlignment":"left"},"content":[],"children":[]}]
```


This is a sample output json for the text editor; make sure to preserve the type, props, and contents

When the frontend receives this information it should incorporate the type, props, and contents when rendering the block